src/components/ReadingProgressPage/ReadingHistory/ReadingStats.tsx

```diff
@@ -53,7 +53,11 @@ const ReadingStats: React.FC<ReadingStatsProps> = ({ activityDay }) => {
         </p>
         <p>
           <ClockIcon />
-          {secondsToReadableFormat(activityDay.secondsRead, t, lang)}
+          {secondsToReadableFormat(
+            activityDay.secondsRead + (activityDay.manuallyAddedSeconds || 0),
+            t,
+            lang,
+          )}
```

Comment:
Nit: I believe this should be moved to a constant e.g. `const secondsRead = activityDay.secondsRead + (activityDay.manuallyAddedSeconds || 0)` instead of inside the component rendering syntax itself.